docs/calico-with-docker/without-docker-networking/ManualSetup.md

```diff
@@ -18,8 +18,14 @@ the example.
 
 ## Requirements
 
-You will need 2 servers (bare metal or VMs) with a  modern 64-bit Linux OS
-and IP connectivity between them.
+You will need 2 servers (bare metal or VMs) with a modern 64-bit Linux OS and
+IP connectivity between them.  The servers must not have any software running
+that tries to manage or interfere with new interfaces that appear (for example
+adding or removing routes or iptables through them, or modifying interface
+state), as that would fundamentally conflict with Calico's programming.  We
```

Comment:
"would fundamentally conflict with Calico's programming".  

Perhaps "is likely to conflict with Calico's programming of iptables and ipsets, leading to loss of connectivity with endpoints"?